File: C/mycps/1stSem/prime.c
#include<stdio.h>

int main()
{
	int n,i,count;

	printf("Enter a number:");

	scanf("%d",&n);

	for(i=1; i<=n; i++)
	{
		if(n%i==0)
		    count += 1;	
	}

	if(count>2)
		printf("It's a Composite no.\n");

	else
		printf("It's a Prime no.\n");
}

File: Java/Assignment 2/Prime.java
import java.util.Scanner;

public class Prime 
{
    public static boolean PrimeCheck(int num) 
    {
        if (num <= 1) 
            return false;
        for (int i = 2; i <= Math.sqrt(num); i++) 
        {
            if (num % i == 0)
                return false;
        }
        return true;
    }

    public static void main(String[] args) 
    {
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter a number: ");
        int number = sc.nextInt();
        
        if (PrimeCheck(number))
            System.out.println(number + " is a Prime Number.");
        else
            System.out.println(number + " is NOT a Prime Number.");
    }
}
